docs/introduction.md

```diff
@@ -2,31 +2,40 @@
 slug: /
 ---
 
-# Introduction
+# Welcome to YOURLS
 
-## Overview
+## Introduction
 
-**YOURLS** is a set of PHP scripts that will allow you to run **Y**our **O**wn **URL** **S**hortener, on your server, using the domain or subdomain of your choice.
+**YOURLS** is a powerful set of PHP scripts that empowers you to run your very own URL shortener on your server. With YOURLS, you have complete control over your links, detailed statistics, plugin support, and more, all wrapped in a free and open-source package.
```

Comment:
A lot of this looks pretty good to me, but let's not delete the origin of "YOURLS" as a name.